src/Compilers/Core/Portable/Operations/IConstructorBodyOperation.cs

```diff
@@ -18,12 +18,11 @@ namespace Microsoft.CodeAnalysis.Operations
     public interface IConstructorBodyOperation : IMethodBodyBaseOperation
     {
         /// <summary>
-        /// Local declarations contained within the <see cref="Initializer"/>.
+        /// Local declarations contained within the <see cref="Initializer" />.
```

Comment:
Hmm, did we have a whitespace in the file name before?

Reply:
Yes. Yes we did.

Reply:
🤦‍♂️